katsdpdisp/data.py

```diff
@@ -743,11 +743,11 @@ def pc_load_letter(self, filename, rows=None, startrow=0, cf=1.8e9, nc=512):
 
             self.add_data2(t*1000, d, np.zeros(d.shape,dtype=np.uint8) , np.arange(len(bls_ordering)), timeseries, timeseriesabs, timeseriesvar, perc, percfl, blmx, blmxfl, flagfraction, 0)
             sys.stdout.flush()
-        print "\nLoad complete..."
+        print("\nLoad complete...")
 
 def subsample(data, sample_size):
     samples = list(zip(*[iter(data)]*sample_size))   # use 3 for triplets, etc.
-    return map(lambda x:sum(x)/float(len(x)), samples)
+    return np.array(list(map(lambda x:sum(x)/float(len(x)), samples)),dtype=np.complex)
```

Comment:
`list(map(lambda` isn't very Pythonic. It's cleaner to do this instead:
```suggestion
    return np.array([sum(x) / float(len(x)) for x in samples], dtype=np.complex)
```